src/extra/normalize.src.css

```
@@ -15,7 +15,7 @@
 
 :where(html) {
   block-size: 100%;
-  font-family: var(--font-sans);
+  font-family: system-ui;
```

```suggestion
  font-family: var(--font-system-ui);
```
jit-props will make sure the only one in the bundle is this value